website/project/model.py

```diff
@@ -3729,3 +3810,125 @@ def _on_reject(self, user, token):
             },
             auth=Auth(user),
         )
+
+
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+    def _on_reject(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+
+class DraftRegistration(AddonModelMixin, StoredObject):
+
+    is_draft_registration = True
```

Comment:
I think it's dead code from a past implementation of draft registrations. Other than here and on the `Node` object, the string `is_draft_registration` does not appear.

I'll remove them both.